web/src/io/getPerCountryCasesData.ts

```diff
@@ -42,13 +17,60 @@ export interface PerCountryCasesData {
   perCountryCasesIntroContent: string
 }
 
-export function getPerCountryCasesDataRaw(): PerCountryCasesDataRaw {
-  return perCountryCasesDataJson as PerCountryCasesDataRaw
+const perCountryCasesDistributionDatum = z.object({
```

Comment:
(Nitpicking: I always found it useful to call those zod things "schema", e.g. `perCountryCasesDistributionDatumSchema`. Maybe a bit lengthy, but the name makes it obvious that it's not an object containing data)